src/javascript/app/pages/trade/purchase.js

```diff
@@ -225,7 +225,6 @@ const Purchase = (() => {
         if (show_chart) {
             const request = {
                 proposal_open_contract: 1,
-                contract_id           : receipt.contract_id,
```

Comment:
global subscription should be made only one time. It should not be done for each purchase of contract

Reply:
@balakrishna-binary thank you, let me fix it👌

Reply:
@balakrishna-binary I've fixed